automatic/sql-server-data-tools/sql-server-data-tools.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>sql-server-data-tools</id>
+    <title>Microsoft SQL Server Integration Services Projects</title>
+    <version>3.0</version>
+    <authors>Microsoft</authors>
+    <owners>BBT Software AG</owners>
+    <projectUrl>https://marketplace.visualstudio.com/items?itemName=SSIS.SqlServerIntegrationServicesProjects</projectUrl>
+    <packageSourceUrl>https://github.com/bbtsoftware/chocolatey-packages/tree/master/automatic/sql-server-data-tools</packageSourceUrl>
+    <!-- todo: icon -->
+    <iconUrl>https://cdn.jsdelivr.net/gh/bbtsoftware/chocolatey-packages@eca9d801fec67aa11df1af98be470d9ab601c5cf/icons/dotnet.png</iconUrl>
```

Comment:
Wrong icon (icon file is also missing). There's also a hash in the URL, therefore URL would still be wrong if file is commited. But I think we have a script in place to update this.